perf/linear_eq_tabular.txt

```diff
@@ -0,0 +1,22 @@
+│ Row │ variable │ Array       │ Contractor  │ MArray      │ SArray1     │ SArray2     │
+├─────┼──────────┼─────────────┼─────────────┼─────────────┼─────────────┼─────────────┤
+│ 1   │ n = 1    │ 1.1426e-5   │ 2.4173e-5   │ 1.5803e-5   │ 1.014e-5    │ 9.512e-6    │
+│ 2   │ n = 2    │ 2.4413e-5   │ 4.0504e-5   │ 3.9829e-5   │ 2.4083e-5   │ 2.4083e-5   │
+│ 3   │ n = 3    │ 4.5194e-5   │ 6.4343e-5   │ 9.1236e-5   │ 4.5533e-5   │ 5.1221e-5   │
+│ 4   │ n = 4    │ 7.2639e-5   │ 9.3671e-5   │ 0.00016058  │ 7.0972e-5   │ 6.9937e-5   │
+│ 5   │ n = 5    │ 0.000103281 │ 0.000127489 │ 0.000262814 │ 0.000104823 │ 0.000102729 │
+│ 6   │ n = 6    │ 0.000141315 │ 0.000169992 │ 0.000416266 │ 0.000144021 │ 0.000139086 │
+│ 7   │ n = 7    │ 0.000195001 │ 0.000217059 │ 0.000680848 │ 0.000198164 │ 0.000191145 │
+│ 8   │ n = 8    │ 0.000247193 │ 0.00027535  │ 0.00122559  │ 0.000251089 │ 0.000241972 │
+│ 9   │ n = 9    │ 0.000306262 │ 0.000338028 │ 0.0020223   │ 0.000310961 │ 0.000299153 │
+│ 10  │ n = 10   │ 0.00037781  │ 0.000414073 │ 0.00294134  │ 0.000386235 │ 0.000367335 │
+│ 11  │ n = 11   │ 0.000465017 │ 0.000489114 │ 0.00585246  │ 0.000473425 │ 0.000447176 │
+│ 12  │ n = 12   │ 0.00055403  │ 0.000587152 │ 0.0110894   │ 0.000573395 │ 0.00054274  │
+│ 13  │ n = 13   │ 0.000659994 │ 0.000692402 │ 0.0160662   │ 0.000686098 │ 0.000643391 │
+│ 14  │ n = 14   │ 0.000777718 │ 0.000793626 │ 0.0184595   │ 0.000800443 │ 0.000767576 │
+│ 15  │ n = 15   │ 0.000910174 │ 0.000952207 │ 0.0239422   │ 0.000972855 │ 0.000899402 │
+│ 16  │ n = 16   │ 0.00107785  │ 0.00111443  │ 0.0303295   │ 0.00113467  │ 0.0010867   │
+│ 17  │ n = 17   │ 0.00122326  │ 0.00125759  │ 0.0398127   │ 0.00134961  │ 0.00125675  │
+│ 18  │ n = 18   │ 0.00167176  │ 0.00159463  │ 0.043795    │ 0.00181409  │ 0.00162584  │
+│ 19  │ n = 19   │ 0.0018632   │ 0.00185617  │ 0.0549107   │ 0.00208529  │ 0.00196587  │
+│ 20  │ n = 20   │ 0.0020604   │ 0.00210595  │ 0.0635598   │ 0.00232704  │ 0.00212374  │
```

Comment:
Actually `n` seems to be redundant since it's just the same as the row.

Reply:
The row numbers need not be sequential due to the stack operations, That's why I kept the column with the value of n